tools/build_dev_container.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -x -e
+
+DOCKER_BUILDKIT=1 docker build \
```

Comment:
We could duplicate this command with the build_args to publish the GPU image. See the ARGS comment below.